comm/base_comm.py

```diff
@@ -255,6 +255,20 @@ def comm_msg(self, stream, ident, msg):
         comm_id = content["comm_id"]
         comm = self.get_comm(comm_id)
         if comm is None:
+            try:
+                from comm import create_comm
+                closed_comm = create_comm(
+                    comm_id=comm_id,
+                    primary=False,
+                    target_name=None,
+                )
+                closed_comm.close()
```

Comment:
We should have some metadata sent back associated with `comm_close` so we are able to identify the source.